### PR TITLE
Add Intro, Participation, Code of Conduct to README

### PR DESCRIPTION
**Pull request by csarven**

### CODE REVIEW

**Review by jaxoncreed — APPROVED** [review]
LGTM

**Comment by jeff-zucker** [comment]
I would like  to see a better statement of what the Solid Team is.  Something along the lines of "The Solid Team, headed by Sir Tim Berners-Lee, provides an organizational umbrella for facilitating a strong open source community in support of Solid.  The Team manages the solidproject.org website, the Solid Forum; provides free Identity and Storage services through solidcommunity.net and fosters outreach efforts such as the "This Month in Solid Newsletter."

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+
+This repository contains organisational work of the Solid Team to meet the needs of the [Solid Project](https://solidproject.org/).
+
+## Participation
+
+All substantive contributors must be members of the Solid Team. It’s easy to [join the Solid Team](https://github.com/solid/process) if you’d like to contribute with any role.
+
+The Solid Team has a (member-only) [chat](https://gitter.im/solid/team).
+
+Every second Wednesday of the month, 16:00 UTC at https://meet.jit.si/solid-team . Meetings are transcribed and [published](https://github.com/solid/team/tree/main/meetings/).
```
Comment (VirginiaBalseiro) [inline README.md:11]: Should this be under a "Meetings" header?

**Review by VirginiaBalseiro — APPROVED** [review]

**Comment by csarven** [comment]
>The Solid Team, headed by Sir Tim Berners-Lee, provides an organizational umbrella for facilitating a strong open source community in support of Solid.

Along those lines sounds good.

As for "the Team manages..", I think we should have separate section for "Work Items", and that can list the team's responsibilities and things of significance..

**Comment by jeff-zucker** [comment]
> Edit: we can follow-up on that last part once Scope stuff is resolved?

Sounds good.

**Pull request merged**